Simplenote/src/androidTest/java/com/automattic/simplenote/BaseUITest.kt

```diff
@@ -49,13 +49,7 @@ open class BaseUITest {
     }
 
     protected fun getRandomString(len: Int): String {
-        val random = SecureRandom()
-        val bytes = ByteArray(len)
-        random.nextBytes(bytes)
-
-        return (bytes.indices)
-                .map { charPool[random.nextInt(charPool.size)] }
-                .joinToString("")
+        return getRandomStringOfLen(len)
```

Comment:
I know this was here before so let's just keep it. However, I don't think it's a good idea to have random values in tests (assuming it is random and not constant. I'm not sure if SecureRandom has a default seed or is indeed random).